rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -197,6 +200,11 @@ def init(
         For example, if you have one application doing object detection
         and another doing camera calibration, you could have
         `rerun.init("object_detector")` and `rerun.init("calibrator")`.
+
+        Application ids starting with `rerun_example_` are reserved for Rerun examples,
+        and will be treated specially by the Rerun Viewer.
+        In particular, it will opt-in to more analytics, and will also
```

Comment:
wasn't aware it's actually "more", thought we only send an additional flag that it was an example and which one?
Not specifying this is here very slightly at odds with our policy of being 100% transparent on analytics

Reply:
We include the application id iff starts with `rerun_example_`, which I count as "more analytics". Maybe we will add other similar things, so I thought it would be better to be broad rather than narrow. I also think it is better to make it sound slightly more scary, so people actually use proper app ids :)